nequip/train/trainer.py

```diff
@@ -850,16 +856,18 @@ def batch_step(self, data, validation=False):
                 self.lr_sched.step(self.iepoch + self.ibatch / self.n_batches)
 
         with torch.no_grad():
-            if hasattr(self.model, "unscale"):
+            if len(self.rescale_layers) > 0:
                 if validation:
-                    # loss function always needs to be in normalized unit
-                    scaled_out = self.model.unscale(out, force_process=True)
-                    _data_unscaled = self.model.unscale(data, force_process=True)
-                    loss, loss_contrib = self.loss(pred=scaled_out, ref=_data_unscaled)
+                    for layer in self.rescale_layers:
+                        # loss function always needs to be in normalized unit
+                        scaled_out = layer.unscale(out, force_process=True)
+                        _data_unscaled = layer.unscale(data, force_process=True)
+                        loss, loss_contrib = self.loss(pred=scaled_out, ref=_data_unscaled)
```

Comment:
this seems wrong? shouldn't it be as above with the `loss` after the loop, and `scaled_out` being updated each iter?

Reply:
```suggestion
                    scaled_out = out
                    _data_unscaled = data
                    for layer in self.rescale_layers:
                        # loss function always needs to be in normalized unit
                        scaled_out = layer.unscale(scaled_out, force_process=True)
                        _data_unscaled = layer.unscale(_data_unscaled, force_process=True)
                    loss, loss_contrib = self.loss(pred=scaled_out, ref=_data_unscaled)
```

Reply:
looks right to me